src/core/socket.c

```diff
@@ -1901,6 +1903,9 @@ static int socket_spawn(Socket *s, ExecCommand *c, PidRef *ret_pid) {
         _cleanup_(exec_params_shallow_clear) ExecParameters exec_params = EXEC_PARAMETERS_INIT(
                         EXEC_APPLY_SANDBOXING|EXEC_APPLY_CHROOT|EXEC_APPLY_TTY_STDIN);
```

Comment:
Hmm, so EXEC_PASS_FDS is currently only used in service_spawn_internal, i.e. has no effect on execute.c/exec-invoke.c. But it feels more correct to set it here.